modules/dns/README.md

```diff
@@ -0,0 +1,68 @@
+# CIS DNS Module
+
+This module is used to create and manage CIS DNS Records.
+
+**Note:**
+
+1. For SRV record, you need to provide record name repeatedly as shown in the below example.
```

Comment:
```suggestion
Provide the SRV record name in both the main and data objects, as shown in the following example.
```

Can you be specific as I've tried here? Where do they need to include the name?

Reply:
It has been eliminated.